### PR TITLE
Add text to ccid protected logo

### PR DESCRIPTION
**Pull request by murny**
#550 

Kinda doesn't work as well in practice, since our thumbnails are 100x100 in size so text becomes hard to read on search result pages...

Previous:
image
image
Changes:
image
image

Using lock icon instead:

image
image

### CODE REVIEW

File: app/views/application/_feature_image.html.erb [hunk 1]
```diff
@@ -7,5 +7,5 @@
     </div>
   <% end %>
 <% else %>
-  <%= image_tag('uofa-logo.jpg', class: 'j-feature-image img-thumbnail',  alt: '', size: '350x350') %>
+  <%= image_tag('uofa-logo.png', class: 'j-feature-image img-thumbnail',  alt: '', size: '350x350') %>
```
Comment (murny) [inline app/views/application/_feature_image.html.erb]: PNG apparently better for text images, half the size too.

File: config/locales/en.yml [hunk 2]
```diff
@@ -420,7 +420,7 @@ en:
       download: 'Download'
       download_all: 'Download All'
       header: 'Files'
-      ccid_restricted_item: Item restricted to University of Alberta users
+      ccid_restricted_item: Item Restricted to University of Alberta Users
```
Comment (murny) [inline config/locales/en.yml]: Make it the same as the image, title case this

**Comment by sfbetz** [comment]
I think this is okay. But, we could also use a nice grey closed padlock icon under the logo instead of the text? That's pretty visual and easy to understand (I think).

**Comment by weiweishi** [comment]
Do we need to get the image from someone? Or @murny would you be able to make this change?

**Pull request merged**